src/components/webview/SourceControlPanelProvider.ts

```diff
@@ -175,6 +205,49 @@ export class SourceControlPanelProvider {
 
 						this.__webviewPanel.reveal(undefined, preserveFocus);
 					}
+
+					if (message.kind === 'webview.sourceControl.createIssue') {
+						const accessToken = this.__userService.getLinkedToken();
+
+						if (accessToken === null) {
+							await routeUserToStudioToAuthenticate();
+							return;
+						}
+
+						const { title, body } = message.data;
+						// call API to create Github Issue
+						const codemodRegistryRepo =
+							'https://github.com/intuita-inc/codemod-registry';
+						const result = await axios.post(
+							'https://telemetry.intuita.io/sourceControl/github/issues',
+							{
+								title,
+								body,
+								accessToken,
```

Comment:
pass the access token as a header called `X-Intuita-Access-Token`, I changed the behavior in the backend PR. In general, authentication/authorization is different than action data

Reply:
will address in a separate PR, because i will remove this whole file